setup.py

```diff
@@ -649,6 +649,7 @@ def pxd(name):
                 'pandas.io.msgpack',
                 'pandas._libs',
                 'pandas.formats',
+                'pandas.plotting',
```

Comment:
the test module was already moved to pandas/tests/plotting before, so no layout changes there I think